homeassistant/components/kodi/media_player.py

```diff
@@ -628,6 +629,23 @@ def media_album_artist(self):
 
         return None
 
+    @property
+    def extra_state_attributes(self) -> dict[str, str | None]:
+        """Return the state attributes."""
+        state_attr: dict[str, str | None] = {}
+        if self.state == MediaPlayerState.OFF:
+            return state_attr
+
+        hdr_type = (
+            self._item.get("streamdetails", {}).get("video", [{}])[0].get("hdrtype")
+        )
+        if hdr_type == "":
+            state_attr["hdr_type"] = "sdr"
```

Comment:
`onkyo` media player has a `dynamic_range` attribute, can we use the same attribute name here?

Reply:
Yes, thanks for the suggestion, sorry for the late response, I was on vacation.